capstone/src/main/webapp/about-us.html

```diff
@@ -33,7 +33,14 @@ <h1 id="club-name"></h1>
           <input type="text" name="announcement-text" placeholder="Add announcement...">
           <input type="submit" value="Post">
       </div>
-      <div id="announcements-display" class="full-screen-panel"></div>
+      <div class="full-screen-panel">
+        <h1>Announcements</h1>
+        <ul id="announcements-display">
+          <template id="announcement-element">
+            <li></li>
```

Comment:
Do you need to include the "li" tags here in order to edit it with the query selector? If so, how will it work when you aren't sure how many announcements there will be? Maybe adding each "li" tag in the for loop in script.js might be better? Not totally sure how templates work though

Reply:
Basically for each element in the JS loop, you select the template, then select something inside the template (in this case I'm selecting the <li> tag), and set something (usually innerHTML) to what you want. Then you can "clone" the template and add it to the end of the parent element. This is just my sense from reading Megan's code and online tutorials - I could be wrong as well.